packages/cozy-client/src/models/document/locales/en.json

```diff
@@ -19,7 +19,7 @@
     "items": {
       "identity": "Identity",
       "family": "Family",
-      "work_study": "Work",
+      "work_study": "Work / Study",
```

Comment:
I'm not a big fan of using this kind of character ('/') here.
These names can be used in file names, and this kind of character will cause some conflicts.
Maybe the word "or" or an underscore ?